packages/make-styles/src/runtime/createCSSVariablesProxy.test.ts

```diff
@@ -17,26 +16,21 @@ describe('createCSSVariablesProxy', () => {
   it('should be able to expand objects containing proxies as properties', () => {
```

Comment:
Maybe we should change the name of this test to something like:

```suggestion
  it('should be able to resolve objects containing proxies as properties', () => {
```